### PR TITLE
Some fixes for Structs and Unions:

### PR DESCRIPTION
**Pull request by goto1134**
This pull request contains fixes for  #72 and  #71  issues and additional array() methods for Struct.

### CODE REVIEW

**Comment by goto1134** [comment]
@henriquemotaesteves review this request, please

**Comment by headius** [comment]
:+1:

We'll wait until your associate reviews the changes. Thanks!

**Comment by henriquemotaesteves** [comment]
@headius, I believe that the request for review has proceeded on the assumption that I was a core commiter. if you do not have objections, please go ahead with the merge process.

Could you please provide to me a feedback about the pull request #46?

**Comment by headius** [comment]
Sorry for the PRs getting a bit stale here, I'll do some work on jnr-* catch-up today.

File: src/test/java/jnr/ffi/struct/UnionTest.java [hunk 1]
```diff
@@ -18,12 +18,11 @@
 
 package jnr.ffi.struct;
 
-import jnr.ffi.Library;
-import jnr.ffi.Runtime;
-import jnr.ffi.TstUtil;
+import jnr.ffi.*;
```
Comment (headius) [inline src/test/java/jnr/ffi/struct/UnionTest.java]: We generally prefer explicit imports up to maybe 10 or 20 in a given package.

Comment (headius) [inline src/test/java/jnr/ffi/struct/UnionTest.java]: We generally prefer explicit imports up to maybe 10 or 20 in a given package.

**Comment by headius** [comment]
We normally don't accept whitespace changes, but yours appear to match what my IDE (IntelliJ) would do and you did them as a separate commit. Kudos!

Thanks for the fixes!

**Pull request merged**